content/vendor/snapshots/snapshots.md

```diff
@@ -0,0 +1,164 @@
+---
+date: 2019-01-23
+linktitle: "Snapshots"
+title: Snapshots
+aliases:
+  - /vendor/snapshots/
+---
+
+## Requirements
+
+### Application YAML
+
+Enable snapshots in your app with the field `spec.allowSnapshots: true` in your kots.io/v1beta1 Application yaml.
+
+```
+apiVersion: kots.io/v1beta1
+kind: Application
+metadata:
+  name: app-slug
+spec:
+  title: App Name
+  allowSnapshots: true
+```
+
+### Customer License
+
+For each customer license there is an Allow Snapshots checkbox. Both the application yaml and the customer license must have snapshots enabled in order for the Admin Console to display snapshot UI to the user.
+
+### Velero
+
+The customer must have [velero](https://velero.io/) installed in the `velero` namespace along with [Restic](https://velero.io/docs/v1.2.0/restic/).
+
+#### Modify default install (this is going away)
```

Comment:
What do you mean "(this is going away)"? When/how is this going away?

Reply:
we're changing kotsadm to be compatible with default velero CLI install